### PR TITLE
Dev guide release post

### PR DESCRIPTION
**Pull request by maelle**
TODOS

- [ ] change links to release version!
- [ ] in the Spanish post, link to Spanish chapters.

### CODE REVIEW

File: content/blog/2024-03-11-devguide-release-0.9.0/index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+This blog post summarises what's new in our Dev Guide 0.9.0, with all changes listed in the [changelog](https://devguide.ropensci.org/booknews.html).
+
+## Now available in Spanish!
+
+Our guide is now bilingual (English and [Spanish](https://devdevguide.netlify.app/es/index.es.html)), thanks to work by [Yanina Bellini Saibene](/author/yanina-bellini-saibene/), [Elio Campitelli](/author/elio-campitelli/) and [Pao Corrales](/author/pao-corrales/), and thanks to [support of the Chan Zuckerberg Initiative, NumFOCUS, and the R Consortium](/blog/2023/01/12/multilingual-publishing-en/).
+[Read the guide in Spanish](https://devdevguide.netlify.app/es/index.es.html).
+
+You can find out more about our multilingual publishing project in the materials and recording from a recent [community call](/commcalls/nov2023-multilingual/).
+Reusable artefacts from the project include our [translation guide](https://translationguide.ropensci.org/), the [babelquarto package](https://docs.ropensci.org/babelquarto/) to render multilingual Quarto books or websites, the [babeldown package](https://docs.ropensci.org/babeldown/) to create and update translations using the DeepL API.
```
Comment (mpadge) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.md:28]:
```suggestion
Useful tools produced by the project include our [translation guide](https://translationguide.ropensci.org/), the [babelquarto package](https://docs.ropensci.org/babelquarto/) to render multilingual Quarto books or websites, and the [babeldown package](https://docs.ropensci.org/babeldown/) to create and update translations using the DeepL API.
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Policy updates
+
+We have updated [Aims and Scope](https://devdevguide.netlify.app/softwarereview_policies#package-categories) to include translation packages, remove experimental text-processing categories, and provide clarifications around API wrappers.
+
+In our [guide for authors](https://devdevguide.netlify.app/softwarereview_author), we now correctly refer to the lifecycle stage "Stable" as the one at which a package can be submitted, thanks to a contribution by GitHub user [bart1](https://github.com/bart1).
+
+## rOpenSci Package maintainer cheatsheet
+
+We have created a [cheatsheet for maintainer of rOpenSci packages](https://devdevguide.netlify.app/maintenance_cheatsheet) that reminds them of how to ask for help on different topics.
```
Comment (mpadge) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.md:41]:
```suggestion
We have created a [cheatsheet for maintainers of rOpenSci packages](https://devdevguide.netlify.app/maintenance_cheatsheet) about how to ask for help on different topics.
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+## Improvements to the packaging guide
+
+The package guide chapter received several improvements.
+
+We've updated our [advice around roxygen2](https://devdevguide.netlify.app/pkg_building#roxygen-2-use), mainly linking to roxygen2 website.
```
Comment (mpadge) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.md:47]:
```suggestion
We've updated our [advice around roxygen2](https://devdevguide.netlify.app/pkg_building#roxygen-2-use), mainly linking to the roxygen2 website. 
```
... and do we want to link to the website there? (Probably not, but wanted to ask regardless.)

File: content/blog/2024-03-11-devguide-release-0.9.0/index.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+We have created a [cheatsheet for maintainer of rOpenSci packages](https://devdevguide.netlify.app/maintenance_cheatsheet) that reminds them of how to ask for help on different topics.
+
+## Improvements to the packaging guide
+
+The package guide chapter received several improvements.
+
+We've updated our [advice around roxygen2](https://devdevguide.netlify.app/pkg_building#roxygen-2-use), mainly linking to roxygen2 website. 
+The advice also includes an explanation of how to document an argument's default, thanks to [Hugo Gruson](https://github.com/bisaloo).
+We have also improved our wording to underline that using roxygen2 still means manual efforts (thanks to [Vincent van Hees](https://github.com/vincentvanhees)).
```
Comment (mpadge) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.md:49]:
```suggestion
We have also improved our wording to emphasise that using roxygen2 still means manual efforts (thanks to [Vincent van Hees](https://github.com/vincentvanhees)).
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+Be respectful and kind! 
+See our reviewers guide and [code of conduct](https://ropensci.org/code-of-conduct/) for more.**
+
+[Alexander Fischer](https://github.com/s3alfisc/) added an explanation of package submission via non-default branches to the [guide for authors](https://devdevguide.netlify.app/softwarereview_author) and to the  [guide for reviewers](https://devdevguide.netlify.app/softwarereview_reviewer#general-guidelines).
+
+Our guide reviewers now clearly explains how to [volunteer as a reviewer, in general and for a particular submission](https://devdevguide.netlify.app/softwarereview_reviewer#volunteering-as-a-reviewer).
+
+## Conclusion
+
+In this post we summarized the changes incorporated into our book “rOpenSci Packages: Development, Maintenance, and Peer Review” over the last months. We are thankful for all contributions that shaped this release.
```
Comment (mpadge) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.md:80]:
```suggestion
In this post we summarized the changes incorporated into our book p“rOpenSci Packages: Development, Maintenance, and Peer Review”](https://devguide.ropensci.org) over the last months.
We are thankful for all contributions that shaped this release. 
```

**Review by mpadge — APPROVED** [review]
Thanks so much for the fabulous work @maelle!

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+slug: r_open_sci_dev_guide_0_9_0_ahora_multilingüe_y_mejor
+tags:
+- Software Peer Review
+- dev guide
+- multilingual
+description: 'Actualizaciones en la versión 0.9.0 del libro en línea ''Paquetes rOpenSci:
+  Desarrollo, mantenimiento y revisión por pares'''
+twitterImg: img/blog-images/2019-05-16-dev-guide-update/cover.png
+output: ~
+twitterAlt: cover of rOpenSci dev guide, showing a package production line with small
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:17]:
```suggestion
twitterAlt: tapa de la guia de desarrollo de rOpenSci que muestra una line ade produccion de paquetes con personas
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+  humans discussing, examining and promoting packages
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:18]:
```suggestion
  discutiendo, examinando y promoviendo paquets. 
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+---
+
+Las orientaciones de la Revisión por Pares de Software rOpenSci se recogen en un [libro en línea](https://devdevguide.netlify.app/booknews) que sigue mejorando.
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md]:
```suggestion
Las orientaciones de la Revisión por Pares de Software de rOpenSci se compilan en un [libro en línea](https://devdevguide.netlify.app/booknews) que sigue mejorando.
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+Esta entrada de blog resume las novedades de nuestra Guía de Desarro 0.9.0, con todos los cambios enumerados en la sección [registro de cambios](https://devguide.ropensci.org/booknews.html).
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:22]:
```suggestion
Esta articulo de blog resume las novedades de nuestra Guía de Desarrollo 0.9.0, con todos los cambios enumerados en la sección [registro de cambios](https://devguide.ropensci.org/booknews.html).
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+
+## ¡Ahora disponible en español!
+
+Nuestra guía es ahora bilingüe (inglés y [español](https://devdevguide.netlify.app/es/index.es.html)), gracias al trabajo de [Yanina Bellini Saibene](/author/yanina-bellini-saibene/), [Elio Campitelli](/author/elio-campitelli/) y [Pao Corrales](/author/pao-corrales/) y gracias a [el apoyo de la Iniciativa Chan Zuckerberg, NumFOCUS y el Consorcio R](/blog/2023/01/12/multilingual-publishing-en/).
+[Lee la guía en inglés](https://devdevguide.netlify.app/es/index.es.html).
+
+Puedes obtener más información sobre nuestro proyecto de publicación multilingüe en los materiales y la grabación de una reciente [convocatoria comunitaria](/commcalls/nov2023-multilenguaje/).
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:29]:
```suggestion
Puedes obtener más información sobre nuestro proyecto de publicación multilingüe en los recursos y video de una [reunión de la comunidad](/commcalls/nov2023-multilenguaje/) reciente.
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+Entre las herramientas útiles producidas por el proyecto se incluyen [guía de traducción](https://translationguide.ropensci.org/es/index.es.html) la [paquete babelquarto](https://docs.ropensci.org/babelquarto/) para representar libros o sitios web Quarto multilingües, y el paquete [paquete babeldown](https://docs.ropensci.org/babeldown/) para crear y actualizar traducciones utilizando la API DeepL.
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:30]:
```suggestion
Entre las herramientas útiles producidas por el proyecto se incluyen la [guía de traducción](https://translationguide.ropensci.org/es/index.es.html) el [paquete babelquarto](https://docs.ropensci.org/babelquarto/) para construir libros o sitios web Quarto multilingües, y el [paquete babeldown](https://docs.ropensci.org/babeldown/) para crear y actualizar traducciones utilizando la API de DeepL.
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+
+Ahora nuestra guía también se traduce a [portugués](https://ropensci.org/blog/2023/10/20/news-october-2023/#help-us-translate-our-dev-guide-to-portuguese) gracias a los voluntarios.
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:32]:
```suggestion
Ahora nuestra guía también se traduce a [portugués](https://ropensci.org/blog/2023/10/20/news-october-2023/#help-us-translate-our-dev-guide-to-portuguese) gracias al esfuerzo de personas voluntarias de nuestra comunidad.
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 13]
```diff
@@ -0,0 +1,85 @@
...
+¡Estamos muy agradecidos por su trabajo!
+
+## Actualizaciones políticas
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:35]:
```suggestion
## Actualizaciones de reglas
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+
+Hemos actualizado [Objetivos y ámbito de aplicación](https://devdevguide.netlify.app/es/softwarereview_policies.es#package-categories) para incluir los paquetes de traducción, eliminar las categorías experimentales de tratamiento de textos y proporcionar aclaraciones sobre las envolturas de API.
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:37]:
```suggestion
Hemos actualizado la sección [Objetivos y ámbito de aplicación](https://devdevguide.netlify.app/es/softwarereview_policies.es#package-categories) para incluir los paquetes de traducción, eliminar las categorías experimentales de tratamiento de textos y proporcionar aclaraciones sobre paquetes que consumen APIs.
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 15]
```diff
@@ -0,0 +1,85 @@
...
+
+En nuestro [guía para autores](https://devdevguide.netlify.app/es/softwarereview_author.es) ahora nos referimos correctamente a la etapa del ciclo de vida "Estable" como aquella en la que se puede enviar un paquete, gracias a una contribución del usuario de GitHub [bart1](https://github.com/bart1).
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:39]:
```suggestion
En nuestra [guía para personas que desarrollan paquetes](https://devdevguide.netlify.app/es/softwarereview_author.es) ahora nos referimos correctamente a la etapa del ciclo de vida "Estable" como aquella en la que se puede enviar un paquete, gracias a una contribución del usuario de GitHub [bart1](https://github.com/bart1).
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 16]
```diff
@@ -0,0 +1,85 @@
...
+
+## Hoja de trucos del mantenedor del paquete rOpenSci
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:41]:
```suggestion
## Hoja de trucos para las personas que mantienen paquetes de rOpenSci
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 17]
```diff
@@ -0,0 +1,85 @@
...
+
+Hemos creado una [para los mantenedores de paquetes rOpenSci](https://devdevguide.netlify.app/es/maintenance_cheatsheet.es) sobre cómo pedir ayuda sobre distintos temas.
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:43]:
```suggestion
Hemos creado una Hoja de trucos para las [personas que mantienen paquetes de rOpenSci](https://devdevguide.netlify.app/es/maintenance_cheatsheet.es) sobre cómo pedir ayuda sobre distintos temas.
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 18]
```diff
@@ -0,0 +1,85 @@
...
+
+## Mejoras en la guía de envasado
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:45]:
```suggestion
## Mejoras en la guía de desarrollo de paquetes
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 19]
```diff
@@ -0,0 +1,85 @@
...
+
+El capítulo de la guía de envases ha recibido varias mejoras.
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:47]:
```suggestion
El capítulo de la guía de desarrollo de paquetes ha recibido varias mejoras.
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 20]
```diff
@@ -0,0 +1,85 @@
...
+
+Hemos actualizado [consejos sobre roxygen2](https://devdevguide.netlify.app/es/pkg_building.es#roxygen-2-use) enlazando principalmente con el [roxygen2](https://roxygen2.r-lib.org/).
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:49]:
```suggestion
Hemos actualizado los [consejos sobre roxygen2](https://devdevguide.netlify.app/es/pkg_building.es#roxygen-2-use) enlazando principalmente con el sitio web de [roxygen2](https://roxygen2.r-lib.org/).
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 21]
```diff
@@ -0,0 +1,85 @@
...
+Los consejos también incluyen una explicación de cómo documentar un argumento por defecto, gracias a [Hugo Gruson](https://github.com/bisaloo).
+También hemos mejorado nuestra redacción para hacer hincapié en que el uso de roxygen2 sigue implicando esfuerzos manuales (gracias a [Vincent van Hees](https://github.com/vincentvanhees)).
+
+Hemos añadido [consejos para paquetes que envuelven APIs web](https://devdevguide.netlify.app/es/pkg_building.es#paquetes-que-envuelven-recursos-web-clientes-api) basados en la entrada del blog ["Por qué deberías (o no deberías) construir un cliente API"](/blog/2022/06/16/publicize-api-client-yes-no/).
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:53]:
```suggestion
Hemos añadido [consejos para paquetes que consumen APIs web](https://devdevguide.netlify.app/es/pkg_building.es#paquetes-que-envuelven-recursos-web-clientes-api) basados en la entrada del blog ["Por qué deberías (o no deberías) construir un cliente API"](/blog/2022/06/16/publicize-api-client-yes-no/).
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 22]
```diff
@@ -0,0 +1,85 @@
...
+
+Nuestras instrucciones sobre [Ficheros de CITACIÓN](https://devdevguide.netlify.app/es/pkg_building.es#citation-file) reflejan ahora las nuevas políticas de CRAN.
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:55]:
```suggestion
Nuestras instrucciones sobre [Ficheros de CITA](https://devdevguide.netlify.app/es/pkg_building.es#citation-file) reflejan ahora las nuevas políticas de CRAN.
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 23]
```diff
@@ -0,0 +1,85 @@
...
+
+Nuestro [lista de paquetes que ayudan en las pruebas](https://devdevguide.netlify.app/es/pkg_building.es#testing) incluye ahora shinytest2 cerca de shinytest, ya que shinytest2 se desarrolla activamente.
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md]:
```suggestion
Nuestra [lista de paquetes que ayudan en las pruebas](https://devdevguide.netlify.app/es/pkg_building.es#testing) incluye ahora shinytest2 cerca de shinytest, ya que shinytest2 se desarrolla activamente.
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 24]
```diff
@@ -0,0 +1,85 @@
...
+
+## Mejoras en la guía de GitHub y CI
+
+Nuestra guía de colaboración incluye más enlaces a [recursos sobre, por ejemplo, revisiones de RP y flujos de trabajo Git](https://devdevguide.netlify.app/es/maintenance_collaboration.es#git-workflow) gracias a [Mauro Lepore](https://github.com/maurolepore).
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md]:
```suggestion
Nuestra guía de colaboración incluye más enlaces a [recursos sobre, por ejemplo, revisiones de PR y flujos de trabajo Git](https://devdevguide.netlify.app/es/maintenance_collaboration.es#git-workflow) gracias a [Mauro Lepore](https://github.com/maurolepore).
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 25]
```diff
@@ -0,0 +1,85 @@
...
+Ese mismo capítulo menciona ahora [Discusiones en GitHub](https://devdevguide.netlify.app/es/maintenance_collaboration.es#communication-with-users) como forma de comunicarse con los usuarios, y más detalles sobre [plantillas de temas](https://devdevguide.netlify.app/es/maintenance_collaboration.es#issue-templates).
+
+[Edward Wallace](https://github.com/ewallace) añadió una explicación de [la integración continua (IC) en general](https://devdevguide.netlify.app/es/pkg_ci.es#what-is-continuous-integration-ci) y de la [codecov en particular](https://devdevguide.netlify.app/es/pkg_ci.es#coverage) a nuestra guía CI.
+
+## Mejor documentación del proceso
+
+Ahora destacamos los valores (el tono) de nuestro sistema en más lugares: en la parte superior de la guía del revisor, en la plantilla para las solicitudes de revisión y en el mensaje que nuestro bot publica después de que los editores [asignan un reviso](https://github.com/ropensci-org/buffy/pull/77/files).
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md]:
```suggestion
Ahora destacamos los valores (el tono) de nuestro sistema en más lugares: en la parte superior de la guía para quienes revisan, en la plantilla para las solicitudes de revisión y en el mensaje que nuestro bot publica después de que quines editan [asignan las personas que revisaran](https://github.com/ropensci-org/buffy/pull/77/files).
```

File: content/blog/2024-03-11-devguide-release-0.9.0/index.es.md [hunk 26]
```diff
@@ -0,0 +1,85 @@
...
+
+> **La comunidad de rOpenSci es nuestro mejor activo.
+> Nuestro objetivo es que las revisiones sean abiertas, no conflictivas y centradas en mejorar la calidad del software.
+> ¡Sé amable!
+> Consulta nuestra guía de revisión y el [código de conducta](https://ropensci.org/code-of-conduct/) para más información.**
+
+[Alexander Fischer](https://github.com/s3alfisc/) añadido una explicación sobre el envío de paquetes a través de ramas no predeterminadas a la página [guía para autores](https://devdevguide.netlify.app/es/softwarereview_author.es) y a la [guía para revisores](https://devdevguide.netlify.app/es/softwarereview_reviewer.es#general-guidelines).
```
Comment (yabellini) [inline content/blog/2024-03-11-devguide-release-0.9.0/index.es.md:75]:
```suggestion
[Alexander Fischer](https://github.com/s3alfisc/) añadido una explicación sobre el envío de paquetes a través de ramas no predeterminadas a la página [guía para autores y autoras](https://devdevguide.netlify.app/es/softwarereview_author.es) y a la [guía para las personas que revisan](https://devdevguide.netlify.app/es/softwarereview_reviewer.es#general-guidelines).
```

**Comment by maelle** [comment]
Muchas gracias @yabellini!!

**Pull request merged**